user-guide/Standard_Apps/Monitoring_Solutions/Performance_Analyzer/Performance_Analyzer.md

```diff
@@ -0,0 +1,26 @@
+---
+uid: Performance_Analyzer
+---
+
+# Performance Analyzer
+
+We often find ourselves in situations where we need to monitor the performance of the solution. The end user will not be satisfied with it unless it’s performant, but how do we track the performance of our code and identify the bottlenecks? Chances are we are forced to implement our own performance tracking, which can be both time consuming and difficult. That’s why we decide to create a generic Performance Analyzer solution that will streamline the process, reduce time to deliver and the cost. 
+
+## Installing the Performance Analyzer
+
+Performance Analyzer is divided in to following: 
+
+## NuGet
+The Performance Analyzer NuGet is designed to track and log performance metrics for methods in single or multi-threaded environments. It provides developers with an easy way to monitor execution times and track method calls across systems by logging performance data to the storage of their choice.
+
+> [!INSTALL]
+> The NuGet can be installed by downloading [Skyline.DataMiner.Utils.PerformanceAnalyzer]().
```

Comment:
link needs to be added

Reply:
I'm still waiting for approval to make the NuGet public.